ros/api/v0/hosts.py

```diff
@@ -187,7 +191,8 @@ def get(self, host_id):
             record['display_performance_score'] = profile.display_performance_score
             record['rating'] = rating_record.rating if rating_record else None
             record['recommendation_count'] = len(system.rule_hit_details)
-            record['state'] = system.rule_hit_details[0].get('key')
+            rule_error_key = system.rule_hit_details[0].get('key')
+            record['state'] = SYSTEM_STATES[rule_error_key]
```

Comment:
As per this [mockup](https://marvelapp.com/prototype/9408d9j/screen/74816603), do we need the state field on the details page?

Reply:
Yeah, as per mockups I think we do not need it in `HostDetailsApi`. I will remove the `state` from here.